babel.config.js

```diff
@@ -1,5 +1,24 @@
-module.exports = {
-  presets: [
-    '@vue/app'
-  ]
-}
+module.exports = api => {
+    const isTest = api.env('test');
```

Comment:
nit: I'm guessing the four spaces for indents is your default, but given the rest of her project uses two spaces for indents, I would stay consistent with that
Same in your spec file

Reply:
this is true, I'll still accept the PR, though, but good looking out!